pkgs/development/interpreters/php/8.0.nix

```diff
@@ -1,11 +1,13 @@
-{ callPackage, lib, stdenv, ... }@_args:
+{ fetchpatch, callPackage, lib, stdenv, ... }@_args:
```

Comment:
```suggestion
{ callPackage, lib, stdenv, ... }@_args:
```

That is unused. Did you try fetching the patch and it did not apply? If so we can remove it again.

Reply:
Yep, I have to remove it. The patch doesn't seems to fix the issue anyway.

Reply:
Fixed.